src/pages/administration/teachers/index.vue

```diff
@@ -20,6 +20,9 @@
 			@update:current-page="onUpdateCurrentPage"
 			@update:rows-per-page="onUpdateRowsPerPage"
 		>
+			<template v-slot:datacolumn-classes="{ data }">
+				<span v-for="c in data" :key="c"> {{ c }} </span>
```

Comment:
```suggestion
				{{ data.join(", ") }}
```
Vllt eher so, um per , zu trennen? (ungetestet)

Reply:
ja läuft